components/logins/src/db.rs

```diff
@@ -201,6 +166,41 @@ fn sqlcipher_3_compat(conn: &Connection) -> Result<()> {
     Ok(())
 }
 
+/// Opens an existing database and fetches the salt.
```

Comment:
It might be worth checking if this is a migration that's already happened? The dynamics of iOS migration might mean they are happier with killing migrations earlier than Fennec is.

Reply:
Thanks for checking on this! According to the channel comments both iOS and android have not killed it yet (even though it's coming soon)